tests.py

```diff
@@ -153,36 +189,37 @@ def fn():
 
     def test_create_authentication_request(self):
         api = toopher.ToopherApi('key', 'secret')
+        pairing_id = str(uuid.uuid4())
```

Comment:
I see that the `authenticate` method can take a username or pairing ID, so the tests were simply updated to follow suit.